src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -694,6 +695,44 @@ public function addProductAction(int $orderId, Request $request): Response
         ]);
     }
 
+    /**
+     * @AdminSecurity("is_granted(['read'], request.get('_legacy_controller'))", redirectRoute="admin_orders_index")
+     *
+     * @param int $orderId
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function refreshProductPricesAction(int $orderId, Request $request): Response
```

Comment:
```suggestion
    public function productPricesAction(int $orderId, Request $request): Response
```

Reply:
Just a suggestion to have a more generic name, because the controller itself doesn't refresh anything it only returns the price list It's the js that uses this data to refresh